codalab/client/local_bundle_client.py

```diff
@@ -12,12 +12,10 @@
 import os
```

Comment:
I'm working on that :)
Currently, code in `legacy.BundleService` still uses it, but I'm looking at how much work it would be to just move all of that into `BundleService` since it's the only user.